errbot/backends/telegram_messenger.py

```diff
@@ -191,9 +191,14 @@ def __init__(self, config):
         self.telegram = None  # Will be initialized in serve_once
         self.bot_instance = None  # Will be set in serve_once
 
-        compact = config.COMPACT_OUTPUT if hasattr(config, 'COMPACT_OUTPUT') else False
-        enable_format('text', TEXT_CHRS, borders=not compact)
-        self.md_converter = text()
+        self.default_send_options = dict(parse_mode='Markdown')
+        self.default_send_options.update(getattr(config, 'TELEGRAM_DEFAULT_SEND_OPTIONS', {}))
```

Comment:
I would like to see `TELEGRAM_DEFAULT_SEND_OPTIONS` added to `config-template.py` with a little bit of accompanying documentation of what can go into it, so that this doesn't become a hidden feature.